src/DifficultyTestsFiller/dfArrowGlacier/difficultyArrowGlacierFromLondonCopier.json

```diff
@@ -0,0 +1,2483 @@
+{
+    "difficultyArrowGlacierFromLondon": {
+        "_info": {
+            "comment": "London->ArrowGlacier gradual difficulty increase test: blocks 13460000 - 15000000, step 5000 blocks, uncle rate 5% "
```

Comment:
what is this test originate from?

Reply:
I filled these test cases externally with a python script, I did not want the filler to make an exhaustive combination so I took a mainnet block as input and start point, and the script filled the steps of the test case where the difficulty gradually increases, to try to simulate the difficulty increase feedback loop in normal block production.

If this is not ok, I think another approach could be to add a filler with combinations on the block where block is supposed to happen.